### ISSUE TITLE
Pinch to zoom

### ISSUE THREAD

**Issue by TheButlah** [wez/wezterm#4184]
**Is your feature request related to a problem? Please describe.**
iTerm2 supports a pinch to zoom. It lets the user easily adjust the size of the font.

**Describe the solution you'd like**
Two-finger-pinch should zoom out, and vice versa. Probably accomplished by changing the font size. It might also change the window size, to keep the number of characters visible the same.

**Additional context**
I'm happy to implement this! If its a feature that would be accepted

**Comment by wez** [comment]
Related to:
* https://github.com/wez/wezterm/issues/792

Note that wezterm has no touch support on any platform.

If you want to add touch related features, there needs to be a design/consideration for how it works on all 4 windowing environments: Windows, macOS, X11 and Wayland.

**Comment by TheButlah** [comment]
To clarify - I mean pinching the trackpad, not the screen

**Comment by wez** [comment]
Yes; both surfaces have touch input and wezterm doesn't have support for touch on any OS yet today.

**Comment by phush0** [comment]
On windows if you use pinch to zoom from trackpad, leads to sporradik jump inside neovim something like super page up or down and it is a very unpleasant experience.

**Comment by Freed-Wu** [comment]
If both support for touchscreen and support for touchpad are same, this issue should be a repetition of #792, so it should be closed as a duplicate.
